### PR TITLE
Add request tracer toggle for httpjson input type

### PR DESCRIPTION
**Pull request by bhapas**
## What does this PR do?

This PR sets a toggle to  `enable_request_tracing` on all httpjson input integrations. It logs the raw requests and responses to a dedicated file. The allows users to collect detailed debugging data that is included in the agent diagnostics dump. 

This also sets minimum kibana version to 8.7.1 as the changes to handle the tracer filename placeholder `*` are incorporated in this version.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related

- https://github.com/elastic/integrations/pull/6115

### CODE REVIEW

File: packages/okta/manifest.yml [hunk 1]
```diff
@@ -32,6 +32,12 @@ policy_templates:
             multi: false
             required: true
             show_user: true
+          - name: enable_request_tracer
+            type: bool
+            title: Enable request tracing
```
Comment (efd6) [inline packages/okta/manifest.yml:37]: Can we have a description as well. Something along these lines
```
The the request tracer logs requests and responses to the agent's local file-system for debugging configurations.
Enabling this request tracing compromises security and should only be used for debugging.
See [documentation](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-httpjson.html#_request_tracer_filename) for details.
```

**Review by andrewkroh — COMMENTED** [review]
I think it would be good to enable the feature in system tests where they exist. Merely exercising the feature could expose critical issues if the feature ever were to trigger something major like a panic. And if someone is debugging system tests that trace log file will already exist for them to analyze.

**Comment by bhapas** [comment]
Breaking this down into multiple PRs to accommodate easy review process

**Comment by bhapas** [comment]
https://github.com/elastic/integrations/pull/6115

**Comment by ishleenk17** [comment]
@bhapas : I see that plan is to segregate the files as per ownership. 
Do we have the PR which is required to be reviewed by obs-service-integrations team ?

**Comment by bhapas** [comment]
> @bhapas : I see that plan is to segregate the files as per ownership. Do we have the PR which is required to be reviewed by obs-service-integrations team ?

@ishleenk17 Not yet... I am fixing the commits... I will put up a PR and link it here and ping your team

File: packages/mimecast/manifest.yml [hunk 2]
```diff
@@ -37,6 +37,14 @@ policy_templates:
             required: true
             show_user: false
             default: https://eu-api.mimecast.com
+          - name: enable_request_tracer
```
[the remark this answers is not in the document]
Comment (bhapas) [reply packages/mimecast/manifest.yml:40]: Fixed in #6115

**Comment by bhapas** [comment]
> > @bhapas : I see that plan is to segregate the files as per ownership. Do we have the PR which is required to be reviewed by obs-service-integrations team ?
> 
> @ishleenk17 Not yet... I am fixing the commits... I will put up a PR and link it here and ping your team

@ishleenk17 #6160 is the PR

**Comment by bhapas** [comment]
https://github.com/elastic/integrations/pull/6161 For obs-cloud-monitoring team

**Comment by bhapas** [comment]
https://github.com/elastic/integrations/pull/6163 for `System` and `Windows` integrations

**Comment by shmsr** [comment]
@bhapas Can you please resolve the merge conflicts and rebase against `main` branch? It'd be helpful when reviewing.

**Comment by bhapas** [comment]
> @bhapas Can you please resolve the merge conflicts and rebase against `main` branch? It'd be helpful when reviewing.

https://github.com/elastic/integrations/pull/6160 is the PR that is broken down from this huge PR.

**Comment by bhapas** [comment]
# 6115 #6160 #6161 #6163 are the broken down PRs.

Closing this big PR to avoid confusion

**Pull request closed without merging**